polaris-core/src/main/java/org/apache/polaris/core/entity/PolarisBaseEntity.java

```diff
@@ -339,10 +339,7 @@ public boolean equals(Object o) {
     PolarisBaseEntity that = (PolarisBaseEntity) o;
     return subTypeCode == that.subTypeCode
         && createTimestamp == that.createTimestamp
-        && dropTimestamp == that.dropTimestamp
-        && purgeTimestamp == that.purgeTimestamp
```

Comment:
Is this change related to the stated purpose of this PR?

Reply:
It was necessary for some test `equals()` comparisons to pass

Reply:
I do not have a strong objection, but the old equals code appears to treat these objects as "value" objects, meaning and difference in any attribute would lead to objects being different. That's just my observation on the code local to this class.

That said, I do not really think these objects are used as value objects by Polaris code other than the related tests, so the change is probably ok.

On the other hand, do test have to compare objects in "whole"? Could ID comparison be sufficient?